expyfun/stimuli/_tracker.py

```diff
@@ -705,6 +705,16 @@ class TrackerDealer(object):
     If dealing from TrackerBinom objects (which is probably not a good idea),
     ``stop_early`` must be ``False`` or else they cannot be ensured to keep
     pace.
+    
+    Selects the tracker from which the next trial should be run
+
+    Returns
+    -------
+    subscripts : list-like
+        The position of the selected tracker.
+    x_current : float
+        The level of the selected tracker.
+    
```

Comment:
This documentation is copied and pasted from the (now deceased) get_trial() function. @Eric89GXL @drammock do you have any thoughts on how to deal with this info?